pkg/cluster/k8sres_test.go

```diff
@@ -1394,7 +1394,7 @@ func TestSidecars(t *testing.T) {
 
 	// replaced sidecar
 	// the order in env is important
-	scalyrEnv := append([]v1.EnvVar{v1.EnvVar{Name: "SCALYR_API_KEY", Value: "abc"}, v1.EnvVar{Name: "SCALYR_SERVER_HOST", Value: ""}}, env...)
```

Comment:
Minor comment: The two added variables don't have to be added to an array just to applied as variadic arguments.

Reply:
Thanks. An oversight. Should be resolved with latest commit.